packages/lib-react-components/src/Markdownz/lib/ping.js

```diff
@@ -12,17 +12,18 @@ export default function plugin({
   ping, // Rename this from pingUsername to ping since we may be pinging more than users
   pingSymbols = ['@'], // Add a parameter to specify what the ping symbols are to be able to support custom ping symbols and more than one.
   resourceURL, // Rename from userURL to resourceURL
-  matchRegex = /@(?:\*\*([^*]+)\*\*|(\w+))/,
+  matchRegex // plugin default is /@([\w\-.]+\b)|#([-\w\d]{3,40})|(\^S[0-9]+)/
 }) {
   if (typeof ping !== 'function' || typeof resourceURL !== 'function') {
     throw new Error(helpMsg)
   }
+  const customMatchRegex = /\B@([\w\-.]+\b)|#([-\w\d]{3,40})|(\^S[0-9]+)/ // match must not be inside an email address
```

Comment:
The original function parameter, `matchRegex` isn't used anymore, so the regex passed in to the function call in the `Markdownz` component isn't used: 

https://github.com/zooniverse/front-end-monorepo/blob/fa4a46044042c00f295a7f94ed7db6905b61988f/packages/lib-react-components/src/Markdownz/Markdownz.js#L120

We should clean this up so it's clear what this plugin is doing. I don't exactly recall why I setup the plugin this way to set the regex by way of function parameter, but I would guess it was to be able to support customization, so this could be handled a few ways:

- Remove the parameter default (already done), set the argument in `Markdownz` ping function call to be the updated `customMatchRegex` and use it by way of the `matchRegex` parameter in the ping plugin. 
- Remove the argument for `matchRegex` in `Markdownz` and set `customMatchRegex` to be the default parameter. Could be done either on line 15:
`matchRegex = /\B@([\w\-.]+\b)|#([-\w\d]{3,40})|(\^S[0-9]+)/`
Or line 20:
`const customMatchRegex = matchRegex || /\B@([\w\-.]+\b)|#([-\w\d]{3,40})|(\^S[0-9]+)/`

`Markdownz` could be updated to have a settings' prop for ping as a future update?